library/lib/view.cpp

```diff
@@ -627,4 +627,13 @@ void View::invalidate(bool immediate)
         this->dirty = true;
 }
 
+View* View::getChildViewAtTouch(float xpos, float ypos){
+    Logger::debug("Touch position (x: %f, y: %f): %s does not provide a getChildViewAtTouch(...) override.", xpos, ypos, describe().c_str());
+    return nullptr;
+}
+
+void View::dragView(float dx, float dy){
+    Logger::debug("Drag performed on %s, but it does nothing.", describe().c_str());
```

Comment:
This will probably be a perf killer :eyes:

Reply:
Yea, I chose to put that there for debugging purposes. I'll make sure to go over all the Logger::debug() calls I added when I finish with the draft.